src/plugins/oauth.js

```diff
@@ -36,8 +36,10 @@ export default {
             this.client.eventEmitter.emit(TOKEN_REFRESHED);
             const tokenRefreshResponse = await res.json();
             await this.client.onRefreshResponse(tokenRefreshResponse);
+            return true;
```

Comment:
Adding this return causes a linter fail. It's a rule I don't really care about, probably some old airbnb stuff, but it doesn't like the else after return. I think it wants this to be like:

```
if (x) {
  ...
  return
}

// here do things that were in else
...
return
```